data/quarg/quarg outfits.txt

```diff
@@ -100,52 +323,190 @@ effect "quarg anti-missile"
 	"velocity scale" 1
 
 
-outfit "Intrusion Countermeasures"
-	plural "Intrusion Countermeasures"
+outfit "Elysian Ranseur"
+	category Guns
+	cost 40000000
+	mass 65
+	thumbnail "outfit/quarg ranseur"
+	"gun ports" -1
+	"outfit space" -65
+	"weapon capacity" -65
+	weapon
+		sound "ranseur"
+		"fire effect" "ranseur"
+		"hit effect" "jump drive"
+		"hit effect" "disruptor impact" 10
+		lifetime 1
+		reload 100
+		velocity 600
+		"firing energy" 900
+		"firing heat" 30
+		"shield damage" 2100
+		"hull damage" 1500
+		"hit force" 600
+		"penetration count" 5
+	description "The technology behind this weapon is incomprehensibly complex, creating pulses of energy that bend spacetime itself. Powerful gravitatonal forces through the opening, piercing through shielding with ease and delivering dangerous blows. How the weapon is able to do any of this is only known to the Quarg."
+
+
+effect "ranseur"
+	sprite effect/ranseur
+		"frame rate" 30
+		"no repeat"
+	lifetime 11
+
+
+outfit "Celestial Culverin"
+	category Guns
+	cost 22000000
+	mass 45
+	thumbnail "outfit/quarg culverin"
+	"gun ports" -1
+	"outfit space" -45
+	"weapon capacity" -45
+	weapon
+		sprite projectile/culverin
+			"frame rate" 60
+			"random start frame"
+			rewind
+		sound "culverin"
+		"hit effect" "culverin hit" 20
+		"die effect" "culverin die"
+		lifetime 110
+		reload 36
+		homing 4
+		velocity 10
+		acceleration 0.6
+		drag 0.05
+		turn 60
+		inaccuracy 12
+		"stream"
+		tracking 1
+		"firing energy" 180
+		"firing heat" 22.5
+		"shield damage" 300
+		"hull damage" 225
+		"slowing damage" 4.5
+		"hit force" -600
+		"missile strength" 600
+	description "This weapon tangles with spacetime in ways beyond understanding, resulting in gravitational forces that slow the target down. As if fulfilling their own intended destiny, the projectiles are guided to their target, and then draw the target back to the Quarg, aiding them in locking down more distant enemies."
+
+effect "culverin hit"
+	sprite "effect/skylance impact"
+		"frame rate" 30
+		"no repeat"
+	lifetime 2
+	"random lifetime" 4
+	"velocity scale" 0
+	"random angle" 180
+	"random velocity" 8
+
+effect "culverin die"
+	sprite "effect/jump drive"
+		"frame rate" 30
+		"no repeat"
+	lifetime 5
+	"velocity scale" 0.4
+
+
+outfit "Astral Ribault"
+	category "Turrets"
+	cost 15000000
+	thumbnail "outfit/quarg ribault"
+	"mass" 35
+	"outfit space" -35
+	"weapon capacity" -35
+	"required crew" 1
+	"turret mounts" -1
+	weapon
+		sprite "projectile/ribault"
+		"hardpoint sprite" "hardpoint/quarg ribault"
+		"hardpoint offset" 10.
+		"fire effect" "ribault flare"
+		"hit effect" "electron impact"
+		sound "ribault"
+		"inaccuracy" 13
+		"turret turn" 2.5
+		"velocity" 13
+		"reload" 17
+		"burst reload" 2
+		"burst count" 19
+		"firing energy" 120
```

Comment:
```suggestion
		"firing energy" 80
```

The Astral Ribault is the only Quarg weapon that's using more than 10 energy per second per ton of space. This drops it down to be more in line with the other Quarg weapons' energy draws, and should also help with the energy issue Arachi raised.